details.md

```diff
@@ -1,15 +1,25 @@
-# Portfolio plans
+# Portfolio Plans
 
-**View portfolio level work items across projects**
+## Portfolio Plans
 
-#### Overview
+**View portfolio level work items across projects.** Portfolio Plans is an organizational tool that helps teams drive cross-project visibility and alignment by tracking portfolio level work in a calendar view.

```

Comment:
How about this...

**View portfolio level work items across projects.** Portfolio Plans is an organizational tool that allows you to pull Epics from any project and team within organization and plots on them on an easy to read timeline view. This helps teams drive cross-project visibility and alignment by tracking portfolio level work.